easybuild/easyconfigs/b/berkeley_upc/berkeley_upc-2.20.2-GCC-4.9.2.eb

```diff
@@ -0,0 +1,34 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+easyblock = 'ConfigureMake'
+
+name = 'berkeley_upc'
```

Comment:
It seems like "Berkeley UPC" is the official name, so how about "BerkeleyUPC"? Not a big fan of underscores in names (and matches what we did with `GlobalArrays`).